css/custom.css

```diff
@@ -175,15 +175,15 @@ nav{
 }
  .has-submenu{
   position: relative;
-} 
+}
```

Comment:
What is this change?

Reply:
I have marked the changes where i did sorry for the trouble